src/set_theory/game/basic.lean

```diff
@@ -495,8 +496,15 @@ inductive inv_ty (l r : Type u) : bool → Type u
 | right₁ : l → inv_ty ff → inv_ty tt
 | right₂ : r → inv_ty tt → inv_ty tt
 
+instance (l r : Type u) [is_empty l] [is_empty r] : is_empty (inv_ty l r tt) :=
+⟨λ h, by { cases h with _ _ _ _ a _ a _; exact is_empty_elim a }⟩
```

Comment:
```suggestion
⟨by rintro (_|_|_|a|a); exact is_empty_elim a⟩
```

Reply:
Nice! This is exactly the sort of proof I was looking for.